README.md

```diff
@@ -47,8 +47,7 @@ The nickel TC is a Kobo variant that mimics Kobo's own TC (as of FW >= 4.6). It
 
 ## Known Issues
 
-Only actively tested on Linux hosts.  
-May not behave properly on Arch Linux, if their ban on static libraries is still in place.  
+Only actively tested on Linux hosts.
```

Comment:
Actually wait a sec, why does this line show? Did the line ending change?

Reply:
Err, what? ^^

Reply:
i deleted some extra spaces at the end of the line by mistake

Reply:
Oh, that's fine. (Except for the fact that GH didn't highlight it.)

Reply:
Oh, yeah, double LF to enforce an actual LF. (Markdown is annoying when you want to place LF yourself :/).

Reply:
Oh, so… not fine? <_<

Reply:
Well, it's a fairly minor nitpick ;).